app/controllers/subscriptions_controller.rb

```diff
@@ -27,6 +28,16 @@ def show
     authorize @subscription
   end
 
+  def activate
+    subscription = Subscription.find_by!(iugu_id: params[:data][:id])
+    authorize subscription
+    subscription.activate!
+    subscription.update!(
+      state: subscription.state,
+      activated_at: subscription.activated_at
```

Comment:
[This line](https://github.com/felipefreitag/pangeas/blob/master/app/models/subscription.rb#L33) already sets `activated_at` when you `activate!`, so you don't need this update as well :)